src/pages/match/DataLineItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useMemo } from 'react';
+
+import Skeleton from '@material-ui/lab/Skeleton';
+
+import Text from '../../components/Text';
+
+export default function DataLineItem({ label, labelId, loading, blank, children })
+{
+  const skeletonWidth = useMemo(() =>
+  {
+    return Math.random() * 80 + 80;
```

Comment:
Why is random variation desired here??

Reply:
Just to make the skeletons look a little better. It was kinda weird when they were all the same size.